gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -603,15 +603,20 @@
   public static final String DEFAULT_REST_SERVER_PORT = "8080";
   public static final String REST_SERVER_ADVERTISED_URI_KEY = "rest.server.advertised.uri";
 
-  /**
+  /*
    * Admin server configuration properties.
    */
   public static final String ADMIN_SERVER_ENABLED_KEY = "admin.server.enabled";
+  /** The name of the class with the admin interface. The class must implement the
+   * AdminWebServerFactory interface .*/
+  public static final String ADMIN_SERVER_FACTORY_CLASS_KEY = "admin.server.type";
```

Comment:
Does the naming of this match how other factories are configured in gobblin?  For instance it seems to make more sense if the key is: `admin.server.factory.type`.